src/families/cosmos/utils.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import { BigNumber } from "bignumber.js";
+import type {
+  CosmosDelegation,
+  CosmosDelegationInfo,
+  CosmosValidatorItem,
+  CosmosFormattedDelegation,
+} from "./types";
+import type { Unit } from "../../types";
+
+export function formatDelegations(
```

Comment:
naming: I wouldn't call this `format` as I wouldn't call the type `CosmosFormatted...`. `format*` function naming we have in live-common is usually for function that return texts.

could we find a better wording?

possible prefix/suffix: Detail, Full, Row, 🤔